app/models/JobExecution.java

```diff
@@ -38,6 +40,7 @@
  * This table have information about the exeuction of the job.
  */
 @Entity
+@EntityConcurrencyMode(ConcurrencyMode.NONE)
```

Comment:
Do we know the root cause of entry for same job execution ID being updated in parallel?

Reply:
@varunsaxena  : Below is the hypothesis (not able to replicate on local machine) 
when we update jobsuggestedparmset , it also updates fitnessJobExecution (which is part of job suggested param set class) ,as all updates are cascading updates . Then later on we also update job-execution separately  . If the fitnessJobExecution and jobexecution points to same object (row) , then it causes OptimisticWriteLockException